tests/test_command_bdist_dmg.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for cx_Freeze.command.bdist_dmg."""
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+from subprocess import run
+
+import pytest
+
+bdist_dmg = pytest.importorskip(
+    "cx_Freeze.command.bdist_dmg", reason="macOS tests"
+).bdist_dmg
+
+if sys.platform != "darwin":
+    pytest.skip(reason="macOS tests", allow_module_level=True)
+
+DIST_ATTRS = {
```

Comment:
This line(s) can be removed.

Reply:
Fixed in eded0cb4268111529354de18cdf6102af46ec1f3